### PR TITLE
Topic Builder matching one in `spine-client`

### PR DESCRIPTION
**Pull request by mdrachuk**
This PR adds a `TopicBuilder` returned by `TopicFactory#select()` to construct subscription topics in a flexible manner.

The `TopicBuilder` API matches that of a `QueryBuilder` introduced in PR #21.

An example of building a `Topic`:
```javascript
requestFactory.topic()
    .select(CUSTOMER_TYPE)
    .byIds(getWestCoastCustomerIds())
    .withMask(["name", "address", "email"])
    .where([
        ColumnFilters.eq("type", "permanent"),
        ColumnFilters.eq("discountPercent", 10),
        ColumnFilters.eq("companySize", Company.Size.SMALL)
    ])
    .build()
```

### CODE REVIEW

File: client-js/src/client/actor-request-factory.js [hunk 1]
```diff
@@ -281,27 +281,38 @@ class Targets {
...
 /**
- * A builder for creating `Query` instances. A more flexible approach to query creation
- * than using a `QueryFactory`.
+ * An abstract base for builders that create `Message` instances which have a `Target`
+ * and a `FieldMask` as attributes.
+ *
+ * <p>The `Target` matching the builder configuration is accessed with `#getTarget()`,
+ * while the `FieldMask` is retrieved with `#getMask()`.
+ *
+ * This classes public API is inspired by the SQL syntax.
```
Comment (armiol) [inline client-js/src/client/actor-request-factory.js]: `classes` does not seem to be correct.

File: client-js/src/client/actor-request-factory.js [hunk 2]
```diff
@@ -281,27 +281,38 @@ class Targets {
...
+ * ```javascript
+ *     select(CUSTOMER_TYPE) // returning <AbstractTargetBuilder> instance
```
Comment (armiol) [inline client-js/src/client/actor-request-factory.js]: I don't understand how the instance of abstract type is returned. I would expect that an instance of descendant is returned.

Let's discuss.

File: client-js/test/client/topic-builder-test.js [hunk 3]
```diff
@@ -0,0 +1,530 @@
...
+import {AnyPacker} from '../../src/client/any-packer';
+import {Message} from 'google-protobuf';
+import {StringValue} from 'spine-web-client-proto/google/protobuf/wrappers_pb';
+import {
+  ColumnFilter,
+  CompositeColumnFilter,
+  EntityFilters
+} from 'spine-web-client-proto/spine/client/entities_pb';
+
+const MILLISECONDS = 1;
```
Comment (armiol) [inline client-js/test/client/topic-builder-test.js]: I don't see how this is used. Let's discuss.

**Comment by mdrachuk** [comment]
@armiol, PTAL.

**Pull request merged**